solutions/security/ai/ease/ease-value-report.md

```diff
@@ -18,3 +18,17 @@ You can interact with the page in the following ways:
 :::{image} /solutions/images/security-ease-value-report.png
 :alt: The Value Report in an EASE project
 :::
+
+## Requirements
+
+```{applies_to}
+serverless: preview
+stack: preview 9.3
+```
+
+To access the **Value report** page, you need the **SOC Management** Security sub-feature [{{kib}} privilege](/deploy-manage/users-roles/cluster-or-deployment-auth/kibana-privileges.md). 
```

Comment:
Maybe we should include that the following roles have the privliege by default: 
Complete: `admin`, `soc_manager`
EASE: `_search_ai_lake_soc_manager`